optimism.tokenlist.json

```diff
@@ -637,6 +637,50 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x3C513dB8Bdc3806e4489d62C3d549A5Aaf6A4e97",
+      "name": "BitBTC",
+      "symbol": "BitBTC",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BitBTC.png",
+      "extensions": {
+        "optimismBridgeAddress": "0xaBA2c5F108F7E820C049D5Af70B16ac266c8f128"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0xc98B98d17435AA00830c87eA02474C5007E1f272",
+      "name": "BitBTC",
+      "symbol": "BitBTC",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BitBTC.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x158F513096923fF2d3aab2BcF4478536de6725e2"
+      }
+    },
+    {
+      "chainId": 42,
+      "address": "0xF6DD2A9B840826D53c9842207628502B79E6B8c2",
+      "name": "BitBTC",
+      "symbol": "BitBTC",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BitBTC.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x0b651A42F32069d62d5ECf4f2a7e5Bd3E9438746"
+      }
+    },
+    {
+      "chainId": 69,
+      "address": "0x83643C9eF0c5707A7815124754d0828c9A38be3a",
+      "name": "BitBTC",
+      "symbol": "BitBTC",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BitBTC.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x0CFb46528a7002a7D8877a5F7a69b9AaF1A9058e"
```

Comment:
Would you mind splitting the PR to allow us to test this on Kovan first

Reply:
Okay, shall we resubmit a PR, or you help us? Thx.

Reply:
Pls submit a PR with just the Kovan entries. Also pls answer these notes https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/45#pullrequestreview-825659719
Happy to chat on Discord or Telegram to move this faster if needed

Reply:
We have resubmit A new [PR 46](https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/46).